packages/jam/transition/disputes/disputes.ts

```diff
@@ -29,6 +29,9 @@ type NewDisputesRecordsItems = {
   toAddToGoodSet: SortedSet<WorkReportHash>;
   toAddToBadSet: SortedSet<WorkReportHash>;
   toAddToWonkySet: SortedSet<WorkReportHash>;
+  toAddToGoodSetDict: HashDictionary<WorkReportHash, true>;
```

Comment:
Couldn't we just use `DisputesRecords` instead of this additional structure? I mean to replace the entire `NewDisputesRecordsItems` with just `DisputesRecords`. The latter has better API and is immutable, here we can easily go out-of-sync with the two?